packages/framework/src/Component/Bytes/BytesHelper.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Bytes;
+
+use function intval;
+
+class BytesHelper
+{
+    /**
+     * @return int
+     */
+    public static function getPhpMemoryLimitInBytes(): int
+    {
+        $memoryLimit = ini_get('memory_limit');
+
+        if ($memoryLimit === '-1') {
+            return -1;
+        }
+
+        return self::convertPhpStringByteDefinitionToBytes($memoryLimit);
+    }
+
+    /**
+     * @param string $memoryLimit
+     * @return int
+     */
+    public static function convertPhpStringByteDefinitionToBytes(string $memoryLimit): int
```

Comment:
can you please add a small unit test for this method? 😇

Reply:
added